app/models/information.rb

```diff
@@ -0,0 +1,21 @@
+# == Schema Information
+#
+# Table name: information
+#
+#  id            :bigint           not null, primary key
+#  section_title :string           not null
+#  text          :text
+#
+# Indexes
+#
+#  index_information_on_section_title  (section_title) UNIQUE
+#
+
+class Information < ApplicationRecord
+  validates :section_title, presence: true, uniqueness: true
```

Comment:
`section_title` can be just title. Also, an enum could be better 🤔 instead of a string , I think title can be `about`, `phone_number`, `address`, etc, right?